packages/react/src/TreeView/TreeView.tsx

```diff
@@ -163,9 +167,9 @@ const UlBox = styled.ul<SxProp>`
 
   .PRIVATE_TreeView-item-toggle {
     grid-area: toggle;
-    display: flex;
```

Comment:
I think we should still keep the `display: flex` and `justify-content: center;` to keep the icon centred horizontally. The VRT are failing due to this.

Reply:
Yep, working on that now :+1: